public/docs/ts/latest/tutorial/toh-pt1.jade

```diff
@@ -32,80 +30,83 @@ include ../_quickstart_repo
     .file typings.json
 :marked
   ## Keep the app transpiling and running
-  We want to start the TypeScript compiler, have it watch for changes, and start our server. We'll do this by typing
+  Type the following command:
 
```

Comment:
This is taking a little too much away. It's important for the reader to know why they are entering the command.